balanceCache.ts

```diff
@@ -0,0 +1,39 @@
+import { Prisma } from '@prisma/client'
+import { AddressPaymentInfo, getAddressPaymentInfo } from 'services/addressService'
+import { redis } from './clientInstance'
+import { Payment } from './types'
+
+// ADDRESS:balance
```

Comment:
What's the value of this comment?

Reply:
It's the format of the caching key. In this case it adds not too much because this is easily readable some lines below at:
```
return `${addressString}:balance`
```

but it's more useful in `paymentCache.ts`, where `// ADDRESS:payments:YYYY:MM
`  is way more readable than:

``` 
return `${addressString}:payments:${moment.unix(timestamp).format(PAYMENT_WEEK_KEY_FORMAT)}`
```

...so I decided to leave it in all cache-specific files, for consistency.